src/core/api_client/tests/factories/auth.factory.ts

```
@@ -0,0 +1,34 @@
+
+  static getTokenEndpoint(config: AuthConfig = {}): string {
+    const { baseUrl, realm } = this.createConfig(config);
+    return `${baseUrl}/realms/${realm}/protocol/openid-connect/token`;
+  }
+
+  static getLogoutEndpoint(config: AuthConfig = {}): string {
+    const { baseUrl, realm } = this.createConfig(config);
+    return `${baseUrl}/realms/${realm}/protocol/openid-connect/logout`;
+  }
+
+  static getApiUrl(path: string = ''): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)